hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/InternalOperationService.java

```diff
@@ -67,4 +69,8 @@
      * @return collection of long running operation logs.
      */
     Collection<JsonSerializable> getSlowOperations();
+
+    <E> void invokeOnPartition(String serviceName, Operation op, int partitionId, Callback<E> callback);
```

Comment:
Can you rename them to asyncOnvokeOn... This makes it immediately clear that it is async.